### PR TITLE
Added education partners

### PR DESCRIPTION
**Pull request by ashmitjsg**
This PR adds an Education Partner section with following organizations:
- JECRC
- JIC

### CODE REVIEW

**Comment by sukso96100** [comment]
Do you have logo image with other colors? the background color of sponsor logo area is white. So people would be difficult to find white logos you just uploaded.

File: src/assets/education_partners/jecrc.png
Comment (sukso96100) [inline src/assets/education_partners/jecrc.png:1]: Dark logo please. This won't be much visible on website's light background.

File: src/assets/education_partners/jic.png
Comment (sukso96100) [inline src/assets/education_partners/jic.png:1]: Dark logo please. This won't be much visible on website's light background.

**Review by sukso96100 — CHANGES_REQUESTED** [review]

**Comment by ashmitjsg** [comment]
> Do you have logo image with other colors? the background color of sponsor logo area is white. So people would be difficult to find white logos you just uploaded.

We don't have one yet. I will request the team to get dark logos.

cc: @Aryan20

**Comment by sukso96100** [comment]
@devilkiller-ag Seems like you guys now have dark logo?

https://x.com/UbuConAsia/status/1814227754440634671?t=6q_fzp2yeYECz1yx1yy2Xg&s=19

**Comment by ashmitjsg** [comment]
Yes, but the ones I have are blue. I have incorporated them in this PR.

**Comment by Aryan20** [comment]
@devilkiller-ag Please post a screenshot of the change as a comment.

**Comment by ashmitjsg** [comment]
> @devilkiller-ag Please post a screenshot of the change as a comment.

Here it is:
Screenshot (495)

**Comment by sukso96100** [comment]
Looks good to me now :) merging!

**Pull request merged**